src/CONST.js

```diff
@@ -881,6 +882,258 @@ const CONST = {
 
     TFA_CODE_LENGTH: 6,
     CHAT_ATTACHMENT_TOKEN_KEY: 'X-Chat-Attachment-Token',
+
+    ALL_COUNTRIES: [
+        'Afghanistan',
+        'Aland Islands',
+        'Albania',
+        'Algeria',
+        'American Samoa',
```

Comment:
Might need to get spanish translation of all these 😬

Reply:
👍  I guess, we'll store only english value in storage.
Also, I would recommend adding a test to check that this list has equal number of entries in both english and Spanish translations and compare values at random indexes.

Reply:
Hmm very true, I like those ideas - but also that got me thinking, we only show States in the StatePicker in English, right? Maybe a follow-up could be translating both of those?

Reply:
Ideally, these countries should be in spanish if selected. Do we display states, countries right now? I don't think so

Reply:
As we discussed 1:1 - We don't display countries yet, states so far we only display the state two letter code, which probably doesn't have a translation 😅

Reply:
Made a follow-up for this since I don't want to hold this PR back much longer: https://github.com/Expensify/App/issues/14549